src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyServiceTests.cs

```diff
@@ -385,6 +389,62 @@ await using (var context = InMemoryApplicationDbContext(contentDbContextId))
             }
         }
 
+        [Fact]
+        public async Task CreateMethodology_SlugNotUnique()
+        {
+            var publication = new Publication
+            {
+                Title = "Test publication"
+            };
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var context = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await context.Publications.AddAsync(publication);
+                await context.SaveChangesAsync();
+            }
+
+            await using (var context = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var methodologyVersionRepository = new Mock<IMethodologyVersionRepository>(Strict);
+                methodologyVersionRepository
+                    .Setup(mock => mock.GetLatestPublishedVersionBySlug(publication.Slug))
+                    .ReturnsAsync(new MethodologyVersion());
+
+                var service = SetupMethodologyService(
+                    contentDbContext: context,
+                    methodologyVersionRepository: methodologyVersionRepository.Object);
+
+                var createdMethodology = new MethodologyVersion
+                {
+                    Id = Guid.NewGuid(),
+                    Methodology = new Methodology
+                    {
+                        Id = Guid.NewGuid(),
+                        Slug = "test-publication",
+                        OwningPublicationTitle = publication.Title,
+                        Publications = new List<PublicationMethodology>
+                        {
+                            new()
+                            {
+                                Owner = true,
+                                Publication = publication
+                            }
+                        }
+                    },
+                    Status = MethodologyApprovalStatus.Draft
+                };
+
+                context.Attach(createdMethodology);
+
+                var actionResult = (await service.CreateMethodology(publication.Id)).AssertLeft();
+
+                VerifyAllMocks(methodologyVersionRepository);
+
+                actionResult.AssertBadRequest(SlugNotUnique);
```

Comment:
I don't think this should be a 400, since there isn't actually anything wrong with the request syntax or headers. I think a 409 (Conflict) would be more appropriate here, or 422 (Processable Entity)

Reply:
What you're saying seems right to me, but I'm no HTTP status code expert, so I'll take your word for it.

I think we return 400 in many in other places where there isn't anything wrong with the request syntax or headers, so I believe this problem is spread throughout the codebase. If we were going to fix this, I'd be inclined to fix it everywhere.  And if we're going to do that, I'd prefer to arrive at a solution with the other devs first rather than jam something in for this PR for a single particular status code. If it was quick to change just this instance, I would, but I think it's going to need a fair chunk of code the way we're currently set up.

So if it's cool with you, I'm inclined to leave this PR as is and raise this at the next dev meeting - where we can agree a holistic solution and get it ticketed and prioritised. If you feel strongly about it getting it fixed now, we can canvas other's opinions - if others agree with you I'm happy to fix this instance as part of this PR.

Reply:
@sambiramairelogic by default, .NET outputs validation errors as 400. This arguably falls into the category of validation errors so it seems fine to me as a status code.

However, as you've said, a more strict interpretation of spec would perhaps indicate 409 or even 422 as being more appropriate. Unfortunately, I think because there's so much overlap in the definitions it's pretty unclear if there's really a true answer to this.

I'd personally lean towards 400 to keep things simple and consistent, rather than have to be too precise. Our private APIs aren't consumed publicly so adding more detail here isn't something to really sweat about.

In the frontend, any codes in the 400 range are basically treated as client errors so the precise semantics of the status code don't really matter that much.